migrations/20170314082605_add_radio_table.js

```diff
@@ -0,0 +1,20 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('radios', function(table) {
+    table.increments('id').primary().index();
+    table.string('name').notNullable().index();
+    table.string('stream');
```

Comment:
Wondering if this should be NOT NULL.. Does it make sense to have a radio without a stream?